library/core/src/num/f128.rs

```diff
@@ -471,7 +471,8 @@ impl f128 {
     #[inline]
     #[must_use]
     #[unstable(feature = "f128", issue = "116909")]
-    pub fn is_sign_positive(self) -> bool {
+    #[rustc_const_unstable(feature = "f128", issue = "116909")]
+    pub const fn is_sign_positive(self) -> bool {
```

Comment:
Thanks for including f16 and f128 :) would you mind adding a commented out gate on methods from this feature? E.g.

```rust
    // #[rustc_const_unstable(feature = "const_float_methods", issue = "...")]
    #[rustc_const_unstable(feature = "f128", issue = "116909")]
    pub const fn is_sign_positive(self) -> bool {
```

We just do that to keep things somewhat linked if API falls under >1 gate.